### PR TITLE
Various EDU Cloud updates based on FAQs [HC-1255]

### PR DESCRIPTION
**Pull request by seaxwi**

### CODE REVIEW

File: content/Arduino Cloud/Spaces/Remove-members-from-a-space.md [hunk 1]
```diff
@@ -3,11 +3,21 @@ title: "Remove members from a space"
 id: 4406710694930
 ---
 
+## Roles that can remove members
+
 You can remove members from space depending on your role:
```
Comment (Renat0Ribeir0) [inline content/Arduino Cloud/Spaces/Remove-members-from-a-space.md:8]:
```suggestion
You can remove members from a space depending on your role:
```

File: content/Arduino Cloud/Spaces/Remove-members-from-a-space.md [hunk 2]
```diff
@@ -3,11 +3,21 @@ title: "Remove members from a space"
...
 
 * **Teachers** can remove **Students**
 * **Admins** can remove any member.
 
+## How removed members are affected
+
+* Removed members keep their Arduino accounts
+* Removed members will no longer have access to courses shared with the space.
+* The removed member will keep their data, but will no longer benefit from additional features and limits provided by any **School plan** active with the space. , but may have some features restricted if they exceed the limits of their personal plan. See [If you exceed plan limits after downgrading](https://support.arduino.cc/hc/en-us/articles/4401874212370-If-you-exceed-plan-limits-after-downgrading).
```
Comment (Renat0Ribeir0) [inline content/Arduino Cloud/Spaces/Remove-members-from-a-space.md:17]:
```suggestion
* The removed member will keep their data, but will no longer benefit from additional features and limits provided by any **School plan** active with the space and may have some features restricted if they exceed the limits of their personal plan. See [If you exceed plan limits after downgrading](https://support.arduino.cc/hc/en-us/articles/4401874212370-If-you-exceed-plan-limits-after-downgrading).
```

Comment (seaxwi) [reply content/Arduino Cloud/Spaces/Remove-members-from-a-space.md]: Fixed with ac39b41

**Review by Renat0Ribeir0 — APPROVED** [review]
LGTM 👍

**Pull request merged**